pkg/apis/datadoghq/v1alpha1/const.go

```diff
@@ -17,4 +17,6 @@ const (
 	// ExtendedDaemonSetOldDaemonsetAnnotationKey annotation key used on ExtendedDaemonset in order to inform the controller that old Daemonset's pod.
 	// should be taken into consideration during the initial rolling-update.
 	ExtendedDaemonSetOldDaemonsetAnnotationKey = "extendeddaemonset.datadoghq.com/old-daemonset"
+	// ExtendedDaemonSetRessourceNodeAnnotationKey annotation key used on Node to overwrite the resource allocated to a specific container linked to an ExtendedDaemonset
+	ExtendedDaemonSetRessourceNodeAnnotationKey = "resources.extendeddaemonset.datadoghq.com/%s.%s.%s"
```

Comment:
can we mention what values should be used for `%s.%s.%s` ? (ns, pod, container ?)
also it would be nice to have the feature documented

Reply:
@ahmed-mez done